src/v2/Apps/NewForYou/NewForYouApp.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { FC, useEffect } from "react"
+import { navigate } from "@reach/router"
```

Comment:
Weird, not sure how this got added to the app! We should use our `useRouter` hook, located in `v2/System/Router`. How did you stumble upon this?

Reply:
not sure. probably just a global search for the word router.